src/pages/index.tsx

```diff
@@ -34,9 +34,9 @@ const index = () => {
                 </Button>
               </NextLink>
             )}
-            <NextLink href="/downloads">
+            <NextLink href="/features-beta">
```

Comment:
Maybe change the link to demo?

Reply:
There is already a link for Demo.

Reply:
The demo link is under usemobilebar and the features link isn't... so altogether I have just replaced the download with demo.